src/test/Square.test.js

```diff
@@ -1,17 +1,42 @@
 import React from "react";
-import Enzyme, {mount} from "enzyme";
-import App from "../App"
+import Enzyme, { mount } from "enzyme";
+import Square from "../components/Square";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 
-Enzyme.configure({ adapter: New Adapter() });
+Enzyme.configure({ adapter: new Adapter() });
 
-describe("<Square />", ()=>{
-    it("render a blank <Square /> component",()=>{
-
-    })
-    it("render a <Square /> with an X", ()=>{
-
-    })
-    it("render a <Square /> with an O",()=>{
-
-    })
-})
\ No newline at end of file
+describe("<Square />", () => {
+  it("render a blank <Square /> component", () => {
+    const imgSrc = "blank/img.png";
+    const wrapper = mount(<Square src={imgSrc} />);
+    console.log(wrapper.debug());
+    const props = wrapper.props();
+    expect(props.src).toBe("blank/img.png");
+  });
+  it("render a <Square /> with an X", () => {
+    const imgSrc = "X/img.png";
+    const wrapper = mount(<Square src={imgSrc} />);
+    const props = wrapper.props();
+    expect(props.src).toBe("X/img.png");
```

Comment:
instead of relying on the appearance that the square is an X, there should be some data to indicate the square is an X

Reply:
I need clarification on this. This is checking the src prop of the square. If I added a state value on the square I would be checking a different prop with state value.

Reply:
I would say based on an internal value of it being X or O, the image changes. We should just look at data to determine if it's X or O and not have to rely on what it appears to be.

Reply:
I've made changes.